src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java

```diff
@@ -93,6 +97,9 @@ public class CaseWorkerProfile extends CaseWorkerDomain implements Serializable
     @MappingField(columnName = "Suspended")
     private String suspended;
 
+    @MappingField(columnName = "staff_admin")
```

Comment:
No need to add this. Can you please remove this?  I think you need to add this
`@JsonProperty("staff_admin")`

Reply:
Needful done

Reply:
thanks @prudhvi-maddineni